CHANGELOG.md

```diff
@@ -9,13 +9,13 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [1.54.0] - 2021-12-03
 
-### Fixed
+### Changed
 
-- Fix CRD API version for EKS infrastcture ref in cluster CR template. 
+- Enable the `login` command to create a client certificate for older workload clusters on Azure.
```

Comment:
@whites11 Can we name the release versions, to make "older" more specific?

Reply:
All existing releases are "older". Any new upcoming release will be "newer".
If you want, > 16.0.2 will need this version of kubectl-gs

Reply:
But no such release exist yet